src/main/java/io/ballerina/plugins/idea/runconfig/BallerinaExecutionConfigOptions.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.ballerina.plugins.idea.runconfig;
+
+import com.intellij.execution.configurations.RunConfigurationOptions;
+import com.intellij.openapi.components.StoredProperty;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Stores Ballerina execution configuration options.
+ *
+ * @since 2.0.0
+ */
+public class BallerinaExecutionConfigOptions extends RunConfigurationOptions {
+
+    private final StoredProperty<String> myScriptName =
```

Comment:
Lets not use `my` prefix for instance variable names. Please apply to all the similar places.
```suggestion
    private final StoredProperty<String> scriptName =
```